feat: preset-lib, config-jest lib agnostic, config-jest-react with react init (#83)
Talend/ui-scripts: packages/config-react-jest/README.md

> # Jest config customisation
> 
> To use the jest configuration in your IDE, or to add customisation
> 
> 1. Create an `jest.config.js` at your project root folder
> 2. Make it extend the one from talend preset
> 
> ```javascript
> const defaults = require('@talend/scripts-config-react-jest/jest.config.js');
> 
> module.exports = {
> 	...defaults,
> 
> 	// add/change default config here
> };
> ```

Reviewer: ```suggestion
# Jest config customization

To use the jest configuration in your IDE, or to add customization

1. Create a `jest.config.js` at your project root folder
2. Make it extend the one from Talend preset

```javascript
const defaults = require('@talend/scripts-config-react-jest/jest.config.js');

module.exports = {
	...defaults,

	// add/change default config here
};
```
```
